source/_integrations/workday.markdown

```diff
@@ -17,10 +17,11 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
-The `workday` binary sensor indicates whether the current day is a workday or not. It allows specifying which days of the week will count as workdays and also
-uses the Python module [holidays](https://pypi.org/project/holidays/) to incorporate information about region-specific public holidays. 
+The `workday` binary sensor indicates whether the current day is a workday or not.
 
-This can be used to make automations that act differently on weekdays vs weekends. For example, you could make your bedroom lights turn on (gently) at 7 in the morning if it is a weekday, but wait until 11 if it is a weekend day.
+It allows specifying which days of the week will count as workdays and also uses the Python module [holidays](https://pypi.org/project/holidays/) to incorporate information about region-specific public holidays. 
+
+This can be used to make daily automations that act differently wether it is a working day or not. For example, you could make your bedroom lights turn on (gently) at 7 in the morning if it is a workday, but wait until 11 if it is a non working day.
```

Comment:
```suggestion
This can be used to make daily automations that act differently on workdays than non-workdays. For example, you could make your bedroom lights turn on (gently) at 7 in the morning if it is a workday but wait until 11 if it is a non-working day.
```